.idea/inspectionProfiles/SeaTunnel.xml

```diff
@@ -0,0 +1,19 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
hi @asdf2014, we already have a `maven-checkstyle-plugin` with `checkstyle.xml`  to help us check the code style. Can we use `maven-checkstyle-plugin` to check the inspection?

And, we can use `mvn checkstyle:check` to check code style when we use `mvn-checkstyle-plugin`, how we can check the inspection after add this profile file?

Reply:
We can add following check rule to `checkstyle.xml`
https://checkstyle.sourceforge.io/config_misc.html#Indentation

Reply:
Hi, @leo65535 . Thanks a lot for your comments. The `maven-checkstyle-plugin` is in the `compile` stage after we finished coding, however the `Code Inspection` is earlier than that, and it will remain us while we are coding. Therefore, they are working on different processes to ensure code quality, which is necessary for us.

Reply:
Thanks @asdf2014, just from my side, I prefer to don't add files to `.idea` folder. `maven-checkstyle-plugin` is enough for us.

Reply:
Hah, you can try it out, the Code Inspection is great, it will remind you in real-time while you are coding, and can automatically fix up your code.

Reply:
@leo65535 You are welcome. This is just the beginning, the .idea can do more than you imagine, and I will raise up more follow-up PRs to empower our project.

Reply:
I have the same opinion as leo, but I'm looking forward to your other about `idea` revamps.

Reply:
Hi @CalvinKirs . Thanks for your comment. You can get more details from [here](https://github.com/apache/druid/tree/master/.idea) for now. In future, I will create another PRs to apply these features.

Reply:
+1, don't add profile files to `.idea` folder.